mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-python_version = 3.8
+python_version = 3.9
```

Comment:
Todoist is still on py3.8. I'd target that.

Reply:
Good idea, as it's also the lowest version we target. (Because of 3.8 we also have to use `from typing import Dict` and `Dict[str, int]` instead of just using `dict[str, int]`.)